app/assets/stylesheets/govuk-overrides.scss

```diff
@@ -86,3 +86,7 @@ button.button.secondary {
     background-image: image-url("separator-2x.png");
   }
 }
+
+.gem-c-search--on-white .gem-c-search__submit {
```

Comment:
What's the reason for this override?

Reply:
The button generated by the GUPC instance of the search bar is blue whilst the search bar button across data.gov.uk currently is black. I'm not sure if it makes more sense to just leave it blue, if there's been research done on this etc. The team's in an awkward position where we can't really get design support so I don't want to make any non-trivial changes. If you think I could get away with a blue button I'm happy to chat to the team and scoop this out.

Reply:
One of the principles of our components is that they're isolated, which means we can have confidence that if it looks correct in the component guide, it looks correct everywhere. If we style components in applications like this, it can lead to problems - for example, if the component was updated to use `govuk-text-colour` as the text colour, in your application the text on the button would suddenly become unreadable. That's not likely to happen, but the rule is there to defend against situations that are more plausible.

The other advantage of not styling components in applications is that we get a consistent look across the site. Is there a reason why DGU has subtle differences like this?

Reply:
I can't dispute this. The question has been asked, will report back.